with-s3/package.json

```diff
@@ -23,7 +23,9 @@
     "react-native-web": "~0.20.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2"
+    "@babel/core": "^7.25.2",
+    "typescript": "~5.8.3",
+    "@types/react": "~19.0.10"
```

Comment:
You do get prompted to install these when you run the project, but other projects in this repo seem to have these listed.